scripts/agent_run.py

```
@@ -0,0 +1,244 @@
+    for i in range(retry):
+        try:
+            # ここで実際の API 呼び出しに差し替え可能
+            return prompt.strip()[:1000]
+        except TooManyRequests:
+            time.sleep(backoff * (2 ** i))
+    # 失敗時でもユーザ向けのメッセージで返す
+    return "LLM呼び出しに失敗しました（429過多）。時間をおいて再実行してください。"
+
+# 各エージェントのアウトプットを合成
+
+def run_pipeline(agents: Agents, topic: str, files: List[Tuple[str, str]], verbose: bool = False) -> Dict[str, str]:
```

Ruff (0.14.1)

Unused function argument: `agents`

(ARG001)